ksqldb-tools/src/main/java/io/confluent/ksql/tools/migrations/commands/BaseCommand.java

```diff
@@ -66,8 +89,18 @@ public int runCommand() {
 
   protected abstract Logger getLogger();
 
+  protected String getConfigFile() {
+    if (configFileGlobal != null) {
+      return configFileGlobal;
+    }
+    if (configFileNonGlobal != null) {
+      return configFileNonGlobal;
+    }
+    return null;
+  }
+
   protected boolean validateConfigFilePresent() {
-    if (configFile == null || configFile.equals("")) {
+    if (getConfigFile() == null || getConfigFile().trim().equals("")) {
```

Comment:
This check is missing in `create`

Reply:
Nice eye! I'm about to open another PR with more miscellaneous fixes. I'll slip this fix into that PR.